app/controllers/admin/rebate_forms_controller.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class Admin::RebateFormsController < ApplicationController
+  before_action :set_admin_rebate_form, only: %i[show edit update destroy]
+
+  # GET /admin/rebate_forms
```

Comment:
@Br3nda whats going to go here? is that submitted form data?

Reply:
It's controller for CRUD on rebate when logged into admin.